addon/components/flexberry-table.js

```diff
@@ -0,0 +1,34 @@
+import Ember from 'ember';
+import layout from '../templates/components/flexberry-table';
+import PaginatedControllerMixin from 'ember-flexberry/mixins/paginated-controller';
+
+export default Ember.Component.extend(PaginatedControllerMixin, {
+  layout,
+
+  _columnCount: Ember.computed('header', {
+    get() {
```

Comment:
<https://emberjs.com/api/ember/2.4/namespaces/Ember.computed> dont contains any word about get

Reply:
Did you mean, i should pass 'key' parameter?
"You can pass a hash with two functions, get and set, as an argument to provide both a getter and setter:"
`fullName: Ember.computed('firstName', 'lastName', {
    get(key) {
      return `${this.get('firstName')} ${this.get('lastName')}`;
    },
    set(key, value) {
      let [firstName, lastName] = value.split(/\s+/);
      this.setProperties({ firstName, lastName });
      return value;
    }
  });`

Reply:
you dont use set(), why use get() here if you can just write computed(%property%, function() { return value;})

Reply:
i'll replece it with function(){}